config/jobs/kubernetes-sigs/tejolote/tejolote-presubmits.yaml

```diff
@@ -0,0 +1,40 @@
+presubmits:
+  kubernetes-sigs/tejolote:
+  - name: pull-tejolote-test
```

Comment:
```suggestion
  - name: pull-tejolote-test
    cluster: k8s-infra-prow-build
```

Reply:
@ameukam should we update the other jobs for the other repos to use that as well?

Reply:
Yeah. we should do it for all the other jobs. rule of the thumb is to have prowjobs running on community-owned insfrastructure.

Reply:
ok will review some jobs